spec/classes/install/container_runtime_spec.rb

```diff
@@ -5,7 +5,9 @@
 describe 'k8s::install::container_runtime' do
   let(:pre_condition) do
     <<~PUPPET
-      include k8s
+      class { 'k8s':
```

Comment:
why are you changing this?

Reply:
Because by default the k8s class will include `k8s::install::container_runtime`, which would then promptly collide with the one created by the test when that's using non-standard parameters - like what's done in the added test.

Reply:
ah okay, thats why.